### ISSUE TITLE
Annual quantities displaying under each month

### ISSUE THREAD

**Issue by tqmurdock** [pacificclimate/climate-explorer-frontend#328]
On the "change from baseline" tab, there are bar plots for each month of the year for something that only exists as an annual measure. 

![image](https://user-images.githubusercontent.com/22355389/62001007-a3bc7580-b09a-11e9-9b99-d15281eb4d8b.png)

**Comment by jameshiebert** [comment]
@rod-glover and @corviday may want to weigh in on the best way to hand this, but I would imagine that we can check the dataset metadata for whether it has monthly data and omit the bar graphs if it does not?

**Comment by rod-glover** [comment]
That seems entirely doable. The metadata contains a `timescale` property, which indicates monthly, seasonal, or yearly, and we can examine those for any given dataset. We may want to do generalize: show a given graph only according to the timescale it's available at.

**Comment by tqmurdock** [comment]
Yes please! Using the timescale property that already exists is perfect.

**Comment by rod-glover** [comment]
Show only bar charts for the given timescale, and change the time axis ticks accordingly.
